web/js/notification-manager.js

```diff
@@ -76,6 +91,48 @@ function show(msg,isModal = false,isError = false,showCancel = false,onClose = (
 	else {
 		cancelBtnEl.classList.add("hidden");
 	}
+
+	if (canDismiss) {
+		addEventListeners(isModal, showCancel);
+	}
+	else {
+		okBtnEl.classList.add("hidden");
+		removeEventListeners();
```

Comment:
Is the call to `removeEventListeners()` here needed? I think `hide()` always unbinds them, and I think `hide()` is called above if `show(..)` has been invoked while the notification is already visible.

Reply:
I will remove that method and the related call.